src/lib/mqtt/mqtt.h

```diff
@@ -0,0 +1,68 @@
+#ifndef MQTT_H
+#define MQTT_H
+
+/*
+*
+* Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Burak Karaboga
+*/
+#include <mosquitto.h>
+
+
+
+/* ****************************************************************************
+*
+* Global vars - 
+*/
+extern const char*      mqttHostname;
+extern unsigned short   mqttPortNumber;
+extern unsigned short   mqttKeepAlivePeriod;
+
+
+
+/* ****************************************************************************
+*
+* mqttInit - 
+*/
+extern void mqttInit (
```

Comment:
This could easilt fit on one single line.
If you prefer not to do that, for readability, then put the `(` on a separate line, and align `)` with it:
```
extern void mqttInit 
(
  const char*    _mqttHost,
  unsigned short _mqttPort,
  unsigned short _mqttKeepAlive
￼);